be/src/storage/lake/txn_log_applier.cpp

```diff
@@ -148,6 +151,22 @@ class PrimaryKeyTxnLogApplier : public TxnLogApplier {
         return Status::OK();
     }
 
+    Status apply_alter_meta_log(const TxnLogPB_OpAlterMeta& op_alter_meta) {
+        DCHECK_EQ(_base_version + 1, _new_version);
```

Comment:
Why do we need this check?

Reply:
It shouldn't happen actually, just preventive programming

Reply:
Do you think whether it will happen or not? @sduzh

Reply:
I think this assertion holds true for now, but it may change in the future.